.github/workflows/continous-integration.yml

```diff
@@ -253,8 +253,8 @@ jobs:
       if: matrix.os == 'windows-latest'
       uses: actions/cache@v1
       with:
-        path: ${{ env.CHOCO_CACHE_LOCATION }}
-        key: ${{ runner.os }}-system-${{ hashFiles('system_test_dependencies/windows.txt') }}
+        path: ${{ github.workspace }}\${{ env.CHOCO_CACHE_LOCATION }}
+        key: ${{ runner.os }}-system-${{ hashFiles('system_test_dependencies/windows.txt') }}-${{ secrets.POETRY_CACHE_VERSION }}
```

Comment:
sounds like we could rename this var to not include `POETRY`, but well, ok 👍🏻

Reply:
Oh yes, let me change it :D